### DISCUSSION TITLE
Increase the limit for discussion categories

### DISCUSSION THREAD

**Discussion by serpro69** [community/community#7960] [Discussions] [answered]
I can see that there's a limit of 9 discussion categories per repo. Is there a reason that we can't have more when needed? Or is it a limitation in "free" plans?

Also, when trying to add more categories, the error actually says that max is 10, but still you can only have up to 9:
![image](https://user-images.githubusercontent.com/22973227/142760653-4e9610ae-b29d-4745-b7b0-83478dea2d8b.png)

**Comment by chaojian-zhang** [comment]
It's definitely horrifying to see this warning when I try to create more categories - that makes me wonder what else are limited in Discussion? Maybe we can't have more than 100 discussions per catagory? Or maybe we can't have more than 25 labels? 

Why is this limit there in the first place?

You know what's even worse? I just realized it seemed Github's own Discussion (https://github.com/github/feedback/discussions) has more than 13 catagories - how is that fair? I even paid for Github Pro.

**Reply by serpro69** [reply]
Not really sure what could be the reason for this limit (or in my case why I can't even reach the limit of 10 and stuck with 9), but I'd really like the github team to reconsider this limit and at least increase it if not lift at all. Seeing that this discussion hasn't been checked by the dev team at all, not sure it will be in any near future though.

**Comment by ashleykolodziej** [comment]
I need this functionality as well - I am trying to set up a feedback board for our product using Discussions and I have 12 areas that people can give feedback on.

**Comment by ssbarnea** [comment]
Not being able to have more than 10 categories is a serious limitation. We have a monorepo where we wanted to create a category for each component, and we have about 20 components so far, likely to grow up to 30 in time.

I wonder if there is any string reason for limiting the number of categories and if there are any plugins to increase that limit.

**Comment by James-4u** [comment]
I think there is a string reason for limiting the number of categories.
Increasing categories doesn't mean encouraging our discussion I think.

**Comment by jfullstackdev** [comment]
as for now, you can have the labels instead, so you can still group the categories by filtering it 

![image](https://user-images.githubusercontent.com/47092464/179769138-8e5ab3b9-3b1d-4dc1-86b9-70ed8526f112.png)

**Comment by BekahHW** [comment]
I'd like to resurface this conversation and see if there's anything that can be done to increase the categories. I really love the nested product categories under Feedback in the GH Community discussions, but we don't have access to two of those features.

**Reply by ssbarnea** [reply]
I am quite curious how we can enable the nesting of categories, I suppose we might need to use special naming for them.

**Comment by siminapasat** [comment]
Hi folks 👋 Here to say that we're aware of the issue and it's high on our backlog to address 🙌

**Comment by jayceeday** [comment, marked as answer]
Hello everyone and thank you for your feedback! 

We have now increased the limit to 25 categories so you can go do that now ✨

Would love to hear your feedback on it! And if 25 works well for you, or if you need more.

**Reply by BekahHW** [reply]
Thank you!

**Reply by serpro69** [reply]
This is amazing! Thank you!

**Reply by mavober** [reply]
It would be great if you could enable the sub category features for everyone,  which you are having here for Github Product Feedback. 
![image](https://user-images.githubusercontent.com/84016053/198243619-3333555b-8f0a-4dfe-913f-cb4d616c20d9.png)

**Reply by jayceeday** [reply]
@mavober we are considering and exploring this! Please add your vote/comment to https://github.com/orgs/community/discussions/32323

**Reply by jimmo** [reply]
@jayceeday Thanks for increasing the number of categories. We're also really hoping to be able to use sub-categories too as well as subscribing to individual categories as per https://github.com/orgs/community/discussions/3951#discussioncomment-3879939!

Just a quick FYI though hoping you can send this to the right place, the documentation still says the limit is 10 categories -- both https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions and https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions

**Reply by jimmo** [reply]
I [just noticed](https://github.com/orgs/community/discussions/32323#discussioncomment-5868933) this is now available on our organization! Great feature!

**Reply by rdtechie** [reply]
@jayceeday Our organization is running into issues with the limit of 25. We definitely need more categories. Is there any chance this can get increased to like 50?

**Reply by sidpolisiti** [reply]
@jayceeday Similar need to what others have mentioned. To avoid engineers hopping from repo to repo. We like to atleast have discussions within one Product area in a repo and then in some cases it can exceed 25 to even ~ more than 50. Is there a chance we get higher limit to Categories. Thanks again.

**Comment by ly525** [comment]
@jayceeday thanks for your  increment for the limit, can you please give the more limit to 50. thanks!

**Reply by alambike** [reply]
@jayceeday , we are migrating from another tool to GitHub Discussions at organization level. In the previous tool where we hosted the community we had more than 60 categories. Is there a way to have these 60 categories in GitHub Discussions, in only one repository?

**Comment by subhasish-nextcraft** [comment]
Would love to see the categories increased!

**Comment by shenweiyan** [comment]
Hope to increase the limit on categories - 25 is too few!
